song.py

```diff
@@ -0,0 +1,172 @@
+
+# save strings as variables and then join them together
+# use f-strings to format so the main lyric stays the same but the ending changes
+# at the end of every chunk: I don't know why she swallowed the fly. Perhaps she'll die.
+
+################################################################
+
+# Here are my initial ideas:
+
+# part_verse_1 = "I know an old lady who swallowed a "
+#   full_verse_1 = part_verse_1 + "fly"
+#   return full_verse_1
+
+# def verse2():
+#   part_verse_2 = part_verse_1
+#   end_verse_2 = part_verse_2 + "spider"
+#   return(end_verse_2)
+# print(end_verse_1)
+
+################################################################
+
+# Here I am thinking about how to do the same lyrics but using a dictionary with key and value pairs
+
+# variations = {
+#         "verse_2": ["bird", "How absurd to swallow a bird!"], 
+#         "verse_3": ["cat", "Imagine that, to swallow a cat!"]
```

Comment:
This is a useful line of inquiry. Can you think about how verse 3 relates to verse 2 and so on? Which parts repeat and which do not? Is there a way to express a verse as the previous verse with some additions/modifications? That's a rhetorical question, there are a couple of ways.